creme/activities/forms/config.py

```diff
@@ -68,6 +68,35 @@ def clean(self, *args, **kwargs):
                 )
             )
 
+        if view_day_start != view_day_end:
```

Comment:
Hum if start & end are "7pm", the range has a 0-duration (I presume it would be an issue), isn't it?

Reply:
if the range as 0-duration I assume that is a 24h duration. But an error if start & end are not the default values (00:00) could be a good idea.

Reply:
The same problem can happens for the working hours range too.